v3/src/components/case-tile-common/attribute-format-utils.tsx

```diff
@@ -97,8 +97,10 @@ export function renderAttributeValue(str = "", num = NaN, attr?: IAttribute, key
 }
 
 export const findLongestContentWidth = (attr: IAttribute) => {
-  // include attribute name in content width calculation
-  let longestWidth = Math.max(kMinAutoColumnWidth, measureText(attr.name, kCaseTableHeaderFont))
+  // We do not include attribute name in content width calculation because header text can be split over
+  // two lines and ellided
```

Comment:
Comment should match what the code is actually doing. Also, `ellipsis` has two `L`s, but `elided` only has one. 😉